ModelComposition.dtd

```diff
@@ -1,10 +1,11 @@
-<!ELEMENT ModelComposition (Description,Inputs?,Outputs?,Composition,Algorithm*,Parametersets,
```

Comment:
Please, keep Algorithm*
The first step is to keep the same Algorithm in different language for the different platforms.
Then, we will try to develop an automatic solution.